src/test/java/org/takes/misc/VerboseIteratorTest.java

```diff
@@ -0,0 +1,72 @@
+package org.takes.misc;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableWithSize;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * 
+ * @author marcus.sanchez (sanchez.marcus@gmail.com)
+ * @version $Id$
+ * @since 0.15.1
+ */
+public class VerboseIteratorTest {
+
+	private static final String NO_NEXT_VALUE_ERROR = "Empty Error Message";
+
+	private final List<String> VALID_LIST = Arrays.asList("Accept: text/plain",
```

Comment:
@m4solutions usually we try not to share variables between test methods unless necessary. If you do declare a variable, it needs a javadoc. also, indentation is broken.